tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py35
+envlist = py35, py36
```

Comment:
We may want to add `py38` at some point to this list.  Don't worry about it for the time being.